.github/workflows/deploy.yml

```
@@ -13,14 +13,42 @@ on:
+    needs: build_contracts
     env:
       DEPLOYMENT_CHAIN: ${{ inputs.network }}
       DEPLOYMENT_KERNEL_ADDRESS: ${{ inputs.kernel_address }}
       SLACK_WEBHOOK_URL: "${{ secrets.DEPLOY_SLACK_WEBHOOK_URL }}"
       TEST_MNEMONIC: "${{ secrets.DEPLOY_TEST_MNEMONIC }}"
+      RUST_LOG: info
     steps:
-      - run: make build
-      - run: make deploy
+      - uses: actions/checkout@v4
+      - uses: actions-rs/toolchain@v1
```

actionlint

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)